src/kv/kvtypes.h

```diff
@@ -191,6 +191,8 @@ namespace kv
     virtual void enable_all_domains() {}
     virtual void resume_replication() {}
     virtual void suspend_replication(kv::Version) {}
```

Comment:
Maybe define `f`'s type in `kvtypes.h` same as `View` and `Seqno`. Could also maybe be defined in `types.h` in ePBFT. Unless it does end up switching to `size_t`